src/libcore/fmt/mod.rs

```diff
@@ -298,7 +298,7 @@ impl<'a> Display for Arguments<'a> {
 ///
 /// For more information on formatters, see [the module-level documentation][module].
```

Comment:
We actually _don't_ have any info here either. http://doc.rust-lang.org/nightly/core/fmt/index.html

Should link to `../../std/fmt/index.html`, perhaps?